pkgs/top-level/all-packages.nix

```diff
@@ -33091,6 +33091,7 @@ with pkgs;
     nix_2_3
     nix_2_4
     nix_2_5
+    nix_2_6
```

Comment:
Generally we don't keep old versions around unless there is a specific reason. So we should keep 2.3 but get rid of 2.4 and 2.5.

Reply:
This was discussed in https://github.com/NixOS/nixpkgs/pull/151419#issuecomment-998216870 and @roberth expressed interest in keeping them all around.

I don't particularly care about this, keeping or not is fine with me.